src/stream/TweetStream.ts

```diff
@@ -59,7 +59,7 @@ export class TweetStream<T = any> extends EventEmitter {
     });
 
     this.res.on('data', chunk => {
-      if (chunk === '\r\n') {
+      if (chunk == '\r\n') {
```

Comment:
Thanks for you report :)

I guess it would be cleaner to call chunk.toString() if chunk is an object?

 (@alkihis maybe you know more about this check)

Reply:
`chunk` should be a Buffer object in this context, I don't think using `==` instead of `===` will work here, right? Even if it is, I'm sure an explicit `toString()` is a better choice :) 
ok for me once this change is made